cockroachcloud/export-logs.md

```diff
@@ -179,13 +183,27 @@ Where:
 - `{secret_key}` is your {{ site.data.products.dedicated }} API key. See [API Access](console-access-management.html) for instructions on generating this key.
 
 
-## Limitations
+## {{ site.data.products.dedicated }} log export Frequently Asked Questions (FAQ)
+
+### Is it possible to configure exported logs to be redacted at source?
+
+Logs exported in this fashion retain [`redactable`](/docs/{{site.versions["stable"]}}/configure-logs.html#redact-logs) markers, but are **not** themselves redacted. Based on your needs, you could optionally redact sensitive data in the logs at your end by making use of `redactable` markers, before the logs are consumed by your team.
```

Comment:
"at your end" meaning you need to put another proxy application in between CRDB and whatever "team" was going to consume "your logs", in order to do the redaction? that is more non-trivial than this makes it sound, no?
You could do that with Satori I think, maybe link to the Satori integration page if that is correct? In any case I'd suggest fleshing out "at your end" a bit more in a way that is helpful for decision-making/estimating the complexity of what is involved.